SU-HSDO/suhumsci
# Arch 83 homepage views

# READY FOR REVIEW

# Summary
- Added "Row" paragraph type to put up to 4 paragraphs side by side.
- New View mode for vertical card
- New view display for featured event and featured publication.

# Needed By (Date)
- ASAP

# Urgency
- Medium

# Steps to Test
1. make sure to do a composer update
2. sync the site
3. enable hs_row module
4. run database updates if you haven't after the sync.
5. create a basic page
6. add a row paragraph item
7. add the "Events: Next Upcoming Featured" view & the "Publications: Single Featured" views to the row.
8. you might need to create a publication, and check the box "Promote to front page"
9. review that the result looks similar to the bottom-ish of this design: https://projects.invisionapp.com/share/GMJJYT1NS79#/screens/299456614


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
